src/global/styles/mixins/overlay.scss

```diff
@@ -0,0 +1,26 @@
+@mixin overlayGapFix($radius, $color: var(--sbb-color-white-default)) {
```

Comment:
mixin should be written in kebab case.

generally I'm not sure if we should provide a mixin wich is tightly coupled to an HTML structure, but I guess you did these to avoid duplication for select?

Reply:
Correct, I solved the naming but this mixin might not be necessary anymore.
I'll have to check with Marco to be sure of it

Reply:
In case it's still needed, add a comment to what it does